src/ext.js

```diff
@@ -248,6 +248,22 @@ export default function ext({ translator, automationsEnabled }) {
                   show: (data) => checkShowNavigation(data, 'websiteUrl'),
                   defaultValue: false,
                 },
+                odagLink: {
+                  type: 'string',
+                  ref: 'odagLink',
```

Comment:
Why are you not storing this on `navigation.odagLink` ?

Reply:
Done.